upi/openstack/inventory.yaml

```diff
@@ -15,6 +15,9 @@ all:
       # OpenShift Ingress floating IP address
       os_ingress_fip: '203.0.113.19'
 
+      os_region_name: ''
+      os_networking_type: 'OpenshiftSDN'
```

Comment:
Can you document this field? It's clear in code that the relevant value is `Kuryr`, however I think it would be nice to add a comment here or in the README